Cannot find module 'json-schema-faker'

when i run
```
prism mock https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore-expanded.yaml
```
show me
```
Error: Cannot find module 'json-schema-faker'
    at Object.<anonymous> (~/.nvm/versions/node/v8.9.3/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/lib/mocker/generator/JSONSchemaExampleGenerator.js:5:11)
```

I get the same error when I do: 

```
➜  Stoplight npm install -g @stoplight/prism-cli@alpha
npm WARN deprecated nomnom@1.5.2: Package no longer supported. Contact support@npmjs.com for more info.
npm WARN deprecated bourne@1.1.2: This module has moved and is now available at @hapi/bourne. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
/Users/taylorbarnett/.nvm/versions/node/v10.15.3/bin/prism -> /Users/taylorbarnett/.nvm/versions/node/v10.15.3/lib/node_modules/@stoplight/prism-cli/bin/run
+ @stoplight/prism-cli@3.0.0-alpha.1
added 288 packages from 276 contributors in 44.784s
➜  Stoplight prism mock https://raw.githack.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore-expanded.yaml

Error: Cannot find module 'json-schema-faker'
    at Object.<anonymous> (~/.nvm/versions/node/v10.15.3/lib/node_modules/@stoplight/prism-cli/node_modules/@stoplight/prism-http/lib/mocker/generator/JSONSchemaExampleGenerator.js:5:11)
```

--- a/package.json
+++ b/package.json
@@ -19,9 +19,6 @@
     "test.update": "yarn test.fast --updateSnapshot",
     "release": "lerna version prerelease --preid alpha --conventional-commit"
   },
-  "resolutions": {
-    "jsonpath": "https://github.com/stoplightio/jsonpath#f1c0e9e634da2d45671b7639fa0a99afc807da17"
-  },
   "devDependencies": {
     "@stoplight/types": "^4.0.0",
     "@types/chance": "^1.0.0",

